INN/umbrella-caribbean
# Add in .btn-orange mixin

## Changes

This pull request makes the following changes:

- Adds a `.btn-orange` mixin so we can create an orange button for the Spanish version of the site in the header

![Screen Shot 2020-07-28 at 10 27 37 AM](https://user-images.githubusercontent.com/18353636/88678959-fc16c080-d0bc-11ea-8969-e1d6f66f6d6a.png)

## Why



For #108

## Testing/Questions

Features that this PR affects:

- Any button that wants to use this new class


Questions that need to be answered before merging:

- [ ] Does the spanish button look ok?

Steps to test this PR:

1. Add this to the "About CIJN" widget in the header and make sure it looks ok
```
<a class="btn-orange btn" href="?lang=es">(En) Español</a>
```
